src/components/sections/section.tsx

```diff
@@ -5,6 +5,7 @@ import {ContainerSizingType, SectionItemProps} from './section-utils';
 
 export type SectionProps = PropsWithChildren<{
   withPadding?: boolean;
+  withFullPadding?: boolean;
```

Comment:
I think we should rewrite this to be `padded: 'full' | 'horizontal' | 'horizontal-bottom' | 'bottom' | 'top'` (etc. matching all cases)